app/Support/SiteTemplateService.php

```
@@ -0,0 +1,226 @@
+                'wp-config-sample.php' => <<<PHP
+<?php
+
+define('DB_NAME', 'replace_me');
+define('DB_USER', 'replace_me');
+define('DB_PASSWORD', 'replace_me');
+define('DB_HOST', '127.0.0.1');
+
+define('WP_HOME', 'https://{$domain}');
+define('WP_SITEURL', 'https://{$domain}');
+
+$table_prefix = 'wp_';
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$table_prefix'. (undefined)

(UnusedLocalVariable)